build.gradle.kts

```diff
@@ -0,0 +1,41 @@
+plugins {
+    id("java-library")
+    id("maven-publish")
+
+    val th2PluginVersion = "0.0.4"
+    id("com.exactpro.th2.gradle.base") version th2PluginVersion
+    id("com.exactpro.th2.gradle.grpc") version th2PluginVersion
+    id("com.exactpro.th2.gradle.publish") version th2PluginVersion
+}
+
+allprojects {
+    val version = project.properties["release_version"].toString()
+    val suffix = project.properties["version_suffix"]?.toString() ?: ""
+    this.group = "com.exactpro.th2"
+    this.version = version + if (suffix.isEmpty()) "" else "-$suffix"
```

Comment:
issue: The `version_suffix` property won't be passed in the workflows you use. Probably, we need to remove this part as it does not do anything actually